core/chains/evm/gas/rollups/l1_oracle.go

```diff
@@ -94,6 +94,9 @@ func NewL1GasOracle(lggr logger.Logger, ethClient l1OracleClient, chainType chai
 	}
 
 	switch chainType {
+	// FIXME Merge conflict, is it still valid?
```

Comment:
@matYang @amit-momin hey folks, you are the ones who might have a context on that, could you please help resolve that conflict? It seems that the code in the chainlink repository is completely different from what we have in ccip

Reply:
This is ok to remove too. This got removed in this [commit](https://github.com/smartcontractkit/chainlink/commit/4ac405a6d5e3bc668ee96eac6bdbecc524467fd6#diff-ce43a8d7f35db6ef5690ca2cb0ae0138dc0efb81926ec0442f05db0f20d0d734L57-L58) when the DA oracle code above got added

Reply:
Coolio, removing!